libs/features/mas/.gitignore

```diff
@@ -0,0 +1,3 @@
+stats.json
```

Comment:
do we need a separate .gitignore file?

Reply:
we have things that are not relevant at the root, it is ok to have .gitignore files in the sub folders.